lib/ffmpeg/transcoder.rb

```diff
@@ -48,7 +59,6 @@ def initialize(movie, output_file, options = EncodingOptions.new, transcoder_opt
         raise ArgumentError, "Unknown options format '#{options.class}', should be either EncodingOptions, Hash or String."
       end
 
-      @transcoder_options = transcoder_options
```

Comment:
Why was this removed?

Reply:
This was moved up a few lines so it was accessible to `requires_pre_encode`